src/System.Runtime.Extensions/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,12 @@
+kernel32.dll!ExpandEnvironmentStringsW
```

Comment:
Not sure of your question. 
I updated the UWP list to the latest. Then looked at what violations appear without a baseline at all. Then baselined just those errors (so some baselines shrunk, which is good)
After this goes in, I will chase up all the violations we see in these baselines. We either need to stop using them, or (preferably) get them whitelisted. The baselines must all go away to ship.